lib/pixify.js

```diff
@@ -10,6 +10,7 @@ var preprocess = require('gulp-preprocess');
 var path = require('path');
 var watchify = require('watchify');
 var fs = require('fs');
+require('colors');
```

Comment:
not super important but `chalk` is generally preferred as it doesn't mess with the string prototype (which can be leaky).

https://github.com/chalk/chalk